pykokkos/core/fusion/util.py

```diff
@@ -52,3 +52,28 @@ def visit_keyword(self, node: ast.keyword) -> None:
         # If the name is not mapped, keep the original name
         node.arg = self.name_map.get(key, node.arg)
         return node
+
+def add_parent_refs(classdef: ast.ClassDef) -> ast.ClassDef:
+    """
+    Add references to each node's parent node in classdef (Copied from
+    static.py, can't be imported due to circular import)
```

Comment:
could it be moved to a util file and then imported in both places?

Reply:
Yes, would require some more refactoring. I'll get to this in a later PR